samples/basic/blinky_pwm/boards/nucleo_l073rz.overlay

```diff
@@ -0,0 +1,38 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright (c) 2022 STMicroelectronics
+ */
+
+#include <dt-bindings/pwm/pwm.h>
+
+
+/ {
+	pwmleds {
+		compatible = "pwm-leds";
+
+		green_pwm_led: green_pwm_led {
+			pwms = <&pwm2 1 4 PWM_POLARITY_NORMAL>;
+		};
+	};
+
+	aliases {
+		pwm-led0 = &green_pwm_led;
+	};
+};
+	
+&timers2 {
+	st,prescaler = <10000>;
+	status = "okay";
+
+	pwm2: pwm {
+		status = "okay";
+		pinctrl-0 = <&tim2_ch1_pa5>;
+		pinctrl-names = "default";
+	};
+};
+
+/* conflicting on pin PA5 */
+&spi1 {
+	status = "disabled";
+};
```

Comment:
Not required as this overlay is sample specific and sample doesn't enable `CONFIG_SPI`.